pyworkflow/__init__.py

```diff
@@ -102,42 +101,71 @@ def getViewerScript():
 
 
 class Config:
```

Comment:
@pconesa I think that Config class has grown a bit, making the init.py difficult to read. 
What do you think about moving this class to a config.py file and still importing it from here:
```python
from .config import Config
```